recipe/meta.yaml

```
@@ -10,35 +10,49 @@ source:
   sha256: bfa03788803061c80c005e28fab5d96f465fcf1c7d28a1eefb1631fb4a2ab67f
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<37]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - setuptools
+    - wheel
   run:
-    - python >=3.7
+    - python
     - google-api-core >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
+    - grpcio >=1.33.2,<2.0dev
+    - grpcio >=1.49.1,<2.0dev # [py>=311]
+    - grpcio-status >=1.33.2,<2.0dev
+    - grpcio-status >=1.49.1,<2.0dev # [py>=311]
```

```suggestion
    - google-api-core-grpc >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
```
Sorry, I didn't check yesterday, but we actually provide a package google-api-core-grpc which has he correct dependencies set.

---

these dependencies are not applicable to version `0.6.10`, which is what we need for https://github.com/AnacondaRecipes/google-generativeai-feedstock/pull/1